tests/ssl-opt.sh

```diff
@@ -695,6 +695,17 @@ run_test() {
     CLI_EXPECT="$3"
     shift 3
 
+    # Expected exit code for the server in response to SIGTERM
+    # Expect 1 for gnutls-serv, otherwise default to expecting 0
+    case $SRV_CMD in
+    gnutls-serv\ *)
```

Comment:
`$SRV_CMD` is often `/path/to/gnutls-serv`, or might be something like `gnutls-serv-1.2.3`. `*gnutls-serv*` would work and I don't think it would have false positives.